gitlab-runner/win-values.yaml

```diff
@@ -2,22 +2,23 @@ rbac:
  create: true
  rules:
  - apiGroups: [""]
-   resources: ["pods", "secrets", "configmaps"]
-   verbs: ["get", "list", "watch", "create", "patch", "delete", "update"]
- - apiGroups: [""]
-   resources: ["pods/exec", "pods/attach"]
+   resources: ["pods/attach","pods/exec"]
    verbs: ["create", "patch", "delete"]
  - apiGroups: [""]
-   resources: ["pods/log"]
-   verbs: ["get"]
-   
+   resources: ["pods","services"]
+   verbs: ["get","watch","create", "delete"]
+ - apiGroups: [""]
+   resources: ["configmaps","secrets"]
+   verbs: ["get","create","update","delete"]
+
+concurrent: 30
 runners:
   tags: "windows,kubernetes"
   config: |
     [[runners]]
-      name = "shell executor runner"
       executor = "kubernetes"
-      
+      limit = 30
```

Comment:
What is the difference between concurrent and limit?

Reply:
First it is important to mention that you may specify few runners under - [[[runners]]](https://docs.gitlab.com/runner/configuration/advanced-configuration.html#the-runners-section) .

limit: limits the number of jobs a single runner instance can run simultaneously
concurrent: Limits how many jobs can run concurrently, across all runner instances under [[runners]].

We don't merge runners' YAMLs into a single config because of the tags and K8s secrets and namespace separation

Reply:
This limitation should be as variables of the module.
Add to README explanation about scaling limitations.
Add a task to handle scaling fully.

Reply:
Added